app/api/sessions.py

```diff
@@ -107,7 +107,7 @@ def query(self, view_kwargs):
             session_type = safe_query(self, SessionType, 'id', view_kwargs['session_type_id'], 'session_type_id')
             query_ = query_.join(SessionType).filter(SessionType.id == session_type.id)
         if view_kwargs.get('microlocation_id') is not None:
-            microlocation = safe_query(self, Track, 'id', view_kwargs['microlocation_id'], 'microlocation_id')
+            microlocation = safe_query(self, Microlocation, 'id', view_kwargs['microlocation_id'], 'microlocation_id')
```

Comment:
@niranjan94 @SaptakS 
Shouldn't these endpoints too be added to docs and dredd test? 
`/microlocations/<int:microlocation_id>/sessions/` and the likes. In docs for each Resource we have two endpoints [ GET and POST ] for list and `<Resource>/<int:resource_id>` 's three endpoint [GET, PATCH, DELETE] but we have many more endpoints implemented. Also what about the relationship endpoints?